app/Base/context/NavbarContext.ts

```diff
@@ -11,6 +13,10 @@ export const NavbarContext = createContext<NavbarContextInterface>({
         // eslint-disable-next-line no-console
         console.error('setNavbarVisibility called on NavbarContext without a provider', value);
     },
+    reset: () => {
+        // eslint-disable-next-line no-console
+        console.error('trying to reset view');
```

Comment:
Please provide appropriate message
```suggestion
        console.error('reset called on NavbarContext without a provider');
```